### DISCUSSION TITLE
关于RTL8367S交换芯片驱动的问题

### DISCUSSION THREAD

**Discussion by zhengxiang311019** [coolsnowwolf/lede#11179] [Q&A / 快速问答]
请问lede目前有支持RTL8367S交换芯片驱动的计划吗？目前有个芯片为RK3568的版本，交换芯片用的RTL8367S，目前在RTL8367B驱动基础上做了少量修改，使用非DSA方式工作，目前交换芯片能够识别，但是收包存在少量丢包情况。不知道是驱动适配问题还是硬件本身存在问题，有没有哪位大佬指点一下？

**Comment by zhengxiang311019** [comment]
目前分析排查出来原因了，RK在自己的驱动代码里面有两个延时变量定义，rx_delay和tx_delay，目前调小这两个参数的值可以解决问题。

&gmac0 {
    phy-mode = "rgmii";
    clock_in_out = "output";

    snps,reset-gpio = <&gpio2 RK_PD3 GPIO_ACTIVE_LOW>;
    snps,reset-active-low;
    /* Reset time is 20ms, 100ms for rtl8211f */
    snps,reset-delays-us = <0 20000 100000>;

    assigned-clocks = <&cru SCLK_GMAC0_RX_TX>, <&cru SCLK_GMAC0>;
    assigned-clock-parents = <&cru SCLK_GMAC0_RGMII_SPEED>, <&cru CLK_MAC0_2TOP>;
    assigned-clock-rates = <0>, <125000000>;

    pinctrl-names = "default";
    pinctrl-0 = <&gmac0_miim
             &gmac0_tx_bus2
             &gmac0_rx_bus2
             &gmac0_rgmii_clk
             &gmac0_rgmii_bus>;

    tx_delay = <0x1c>; //old value:0x3c
    rx_delay = <0x1f>; //old value:0x2f

    phy-handle = <&rgmii_phy0>;
    status = "okay";
};

**Reply by zg9uagfv** [reply]
你好，gmac0连接8367s的phy嘛？是走mdio还是smi？
